ios-target/View/Home/HomeViewController.swift

```diff
@@ -52,3 +92,16 @@ extension HomeViewController: HomeViewModelDelegate {
     }
   }
 }
+
+extension HomeViewController: LocationDelegate {
```

Comment:
can't seem to find where are you setting this view controller as the LocationDelegate

Reply:
Also, shouldn't the viewModel be the delegate? otherwise you have half the location info there and half on the view 🤔